apps/rotm/translations/src/en/pages.json

```diff
@@ -53,6 +53,7 @@
     }
   },
   "confirmation": {
+    "header": "Check your report",
```

Comment:
the confirmation page doesn't need a header, so you would have to add the title to the view instead